docs/en/release_notes/release-3.4.md

```diff
@@ -4,6 +4,32 @@ displayed_sidebar: docs
 
 # StarRocks version 3.4
 
+## 3.4.1
+
+Release Date: March 12, 2025
+
+### New Features and Enhancements
+
+- Data lake analytics supports Deletion Vector in Delta Lake.
+- Supports secure views. By creating a secure view, you can prevent users without the SELECT privilege on the referenced base tables from querying the view (even if they have the SELECT privilege on the view).
+- Supports for Sketch HLL ([`ds_hll_count_distinct`](https://docs.starrocks.io/docs/sql-reference/sql-functions/aggregate-functions/ds_hll_count_distinct/)). Compared to `approx_count_distinct`, this function provides higher-precision approximate deduplication.
```

Comment:
use relative link

Reply:
Full links are required in release notes.